### ISSUE TITLE
Optimize property intialization

### ISSUE THREAD

**Issue by VBAndCs** [paul1956/CSharpToVB#25]
You convert this C#:
```C#
public List<int> Foo {get; set} = new List<int>();
```
To:
```VB.NET
Public Property Foo As List(Of Integer) = new List(Of Integer) ( )
```

It can be optimized to:
```VB.NET
Public Property Foo As new List(Of Integer)( )
```

**Comment by paul1956** [comment]
@VBAndCs I could and do sometimes but not everything. I will look into doing it more often.

**Comment by paul1956** [comment]
Fixed in latest build 4.2.0.1.
